util/fhir/dates.ts

```diff
@@ -166,17 +166,18 @@ export function getRandomPeriodInPeriod(start: string, end: string): fhir4.Perio
 }
 
 /**
- * Takes in resource and returns the Date and dateType formatted nicley (used in Resource Display cards)
+ * Takes in resource and returns the Date and dateType formatted nicely (used in Resource Display cards)
  * @param {fhir4.FhirResource} resource resource to have date formatted
  * @param {string} end end date string for the period
  * @returns {Object} two strings  {date, dateType} if date is a period it will be in the format of mm/dd/yyyy-mm/dd/yyyy
```

Comment:
Small change to update the comment return type to match something like:
`@returns {Object}  {date <Date>, dateType  <string>, sortDate <Date>} if date is a period it will be in the format of mm/dd/yyyy-mm/dd/yyyy. sortDate is initially set to startTime if date is a period`

Reply:
or 
```
/**
 * Takes in resource and returns the Date,  dateType and sortDate formatted nicely (used in Resource Display cards)
 * @param {fhir4.FhirResource} resource resource to have date formatted
 * @param {string} end date string for the period
 * @returns {Object}  {date <Date>, dateType  <string>, sortDate <Date>} if date is a period it will be in the format of mm/dd/yyyy-mm/dd/yyyy. sortDate is initially set to startTime if date is a period
 */
```

Reply:
Thanks for catching! There were a couple of futzy things in here, so that was the bump I needed to make a type for the return object.